client/src/screen/LoginPage/LoginPage.js

```diff
@@ -1,30 +1,76 @@
 import FomrInput from "../../components/FormInput/FormInput.vue";
 import Constant from "@/constant/Constant"; // import file contain variables
+import axios from "axios";
 
-export default {
-	name: "LoginPage",
-	components: {
-		FomrInput,
-	},
-	data() {
-		return {
-			routes: [{ name: "Lobby", url: "/lobby" }],
-			inputInfos: [Constant.REGISTER_INFOS[0], Constant.REGISTER_INFOS[1]],
-		};
-	},
+const baseUrl = "http://127.0.0.1:5000";
 
-	methods: {
-		onShow() {
-			// if type = text -> convert type = password and vice_versa
-			if (this.inputInfos[1].type == "text") {
-				this.inputInfos[1].type = "password";
-			} else {
-				this.inputInfos[1].type = "text";
-			}
-		},
-		onLogin() {
-			this.$router.push({ name: "Lobby" });
-		},
-		computed: {},
-	},
+axios.defaults.headers = {
+    "Cache-Control": "no-cache",
+    Pragma: "no-cache",
+    Expires: "0",
 };
+
+export default {
+    name: "LoginPage",
+    components: {
+        FomrInput,
+    },
+    data() {
+        return {
+            routes: [{ name: "Lobby", url: "/lobby" }],
+            inputInfos: [Constant.REGISTER_INFOS[0], Constant.REGISTER_INFOS[1]],
+            userName: "",
+            passWord: "",
+        };
+    },
+    computed: {
+        // enable login button when input field is not empty
+        isDisable() {
+            return !this.userName || !this.passWord;
+        },
+    },
+    methods: {
+        onShow() {
+            // if type = text -> convert type = password and vice_versa
+            if (this.inputInfos[1].type == "text") {
+                this.inputInfos[1].type = "password";
+            } else {
+                this.inputInfos[1].type = "text";
+            }
```

Comment:
1. Số 1 là Mafic number, nên đặt nó 1 cái tên mô tả là 1 const
- Ví dụ: const PASSWORD_INDEX = 1
2. Ngoài ra phần này có thể viết ngắn gọn lại không
Gợi ý dùng toán tử 3 ngôi

Reply:
done